src/Cache/Container/ActiveContainer.php

```diff
@@ -211,16 +211,20 @@ public function get(string $key, Transformation $transformation): string|int|arr
         return $this->buildFinalTransformation($transformation)->transform($unpacked_values);
     }
 
-    public function set(string $key, string|int|array|bool|null $value): void
+    public function set(string $key, string|int|array|bool|null $value, int $ttl = null): void
     {
         if ($this->isLocked()) {
             return;
         }
+        if ($ttl === null) {
```

Comment:
Can we strip that down to a `$ttl = $ttl ?? $this->config->getDefaultTTL();` here?